laravel-app/app/Interfaces/PostServiceInterface.php

```diff
@@ -8,8 +8,17 @@
 interface PostServiceInterface
 {
     /**
-     * @param User $user
-     * @param array<mixed, mixed> $validatedData
+     * To create a new post of the user.
      */
-    public function createPost(User $user, array $validatedData): UserPost;
+    public function createPost(User $user, array $validatedData): UserPost; // @phpstan-ignore-line
```

Comment:
remove @phpstan-ginore-line

Reply:
sir the one you commented is the outdated version of the PR, I already removed them from last 2 commits. Thank you